wg-security-audit/RFP.md

```diff
@@ -0,0 +1,97 @@
+# Request for Proposal
+
+## Kubernetes Third Party Security Audit
+
+The Kubernetes Third-Party Audit Working Group (working group, henceforth) is soliciting proposals from select Information Security vendors for a comprehensive security audit of the Kubernetes Project. 
+
+### Eligible Vendors
+
+Only the following vendors will be permitted to submit proposals:
+
+- NCC Group
+- Trail of Bits
+- Cure53
+- Bishop Fox
+- Insomnia
+- Atredis Partners
+
+### RPF Process
+
+This RFP will be open between 2018/10/22 and 2019/11/18.
+
+The working group will answer questions for the first two weeks of this period. 
+
+Questions can be submitted [here](https://docs.google.com/forms/d/e/1FAIpQLSd5rXSDYQ0KMjzSEGxv0pkGxInkdW1NEQHvUJpxgX3y0o9IEw/viewform?usp=sf_link). All questions will be answered publicly in this document.
+
+Proposals must include CVs, resumes, and/or example reports from staff that will be working on the project.
+
+- 2018/10/22: RFP Open, Question period open
+- 2018/11/05: Question period closes
+- 2018/11/19: RFP Closes
+- 2018/11/26: The working group will announce vendor selection
+
+## Audit Scope
+
+The scope of the audit is the 3 most recent releases (1.10, 1.11, 1.12) of the core [Kubernetes project](https://github.com/kubernetes/kubernetes). 
+
+- Findings within the [bug bounty program](https://github.com/kubernetes/community/blob/master/contributors/guide/bug-bounty.md) scope are in scope.
+
+> We want the focus of the audit to be on bugs on Kubernetes. While Kubernetes relies upon a container runtimes such as Docker and CRI-O, we aren't looking for (for example) container escapes that rely upon bugs in the container runtime (unless, for example, the escape is made possible by a defect in the way that Kubernetes sets up the container).
```

Comment:
This renders as a quote which might be a little confusing. I know you were quoting my email, but I'm fine to just have it in there as part of the body text with any changes you want to make to the wording.

Reply:
@JayBeale I'm amenable to scoping to 1.12. Let's discuss in today's meeting.